README.md

```diff
@@ -9,7 +9,7 @@ This library is focused on embedded and resource-limited systems.
 
 ## Usage examples
 This is a code example showing the serialization and deserialization of a string.
-As *MicroCDR* uses a static buffer, that means the user has to provide a defined buffer and its size during the *MicroBuffer* creation.
+As *MicroCDR* uses a static buffer, that means the user has to provide a defined buffer and its size during the *mcMicroBuffer* creation.
```

Comment:
Adding the namespace in the names, now is not necessary the Micro suffix.

MicroBuffer -> mcBuffer

Reply:
ok